.changeset/selfish-dancers-sort.md

```diff
@@ -0,0 +1,6 @@
+---
+"vercel": minor
+"@vercel-internals/types": minor
```

Comment:
```suggestion
```

Private packages don't need to be versioned

Reply:
@styfle thanks, fixed!